frontend/src/bundles/common/components/icons/icons.tsx

```diff
@@ -0,0 +1,80 @@
+import { createIcon } from '../components.js';
+
+const TemplatesIcon = createIcon({
+    displayName: 'Templates',
+    viewBox: '0 0 18 18',
+    d: 'M2 18c-.55 0-1.02-.196-1.413-.587A1.926 1.926 0 010 16V2C0 1.45.196.98.588.587A1.926 1.926 0 012 0h14c.55 0 1.02.196 1.413.588C17.803.979 18 1.45 18 2v14c0 .55-.196 1.02-.587 1.413A1.926 1.926 0 0116 18H2zm14-5H2v3h14v-3zm-2.5-2H16V2h-2.5v9zM2 11h2.5V2H2v9zm4.5 0h5V2h-5v9z',
+});
```

Comment:
For icons, it is better to use @chakra-ui/icons. If it doesn’t have the necessary icon, you can use https://github.com/FortAwesome. In [this pr](https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/36) you can see the example

Reply:
@o-nedashkivska  @anton-otroshchenko  Hi, thanks for all your feedbacks, really apreciate it, i dont see the same icons neither in @chakra-ui/icons or in [Fort](https://github.com/FortAwesome)

I suggested using the more similar, what do you think:
![image](https://github.com/user-attachments/assets/881ca19b-2bf6-4c0f-b80d-4b589dcd0f95)

Reply:
Looks good to me :)